src/index.html

```diff
@@ -11,7 +11,190 @@
+            min="1"
+            max="100"
+          >
+        </label>
+        <label
+          for="BirthDate"
+          class="label"
+        > Full date of birth:
+          <input
+            type="date"
+            name="birthDate"
+            id="BirthDate"
+          >
+        </label>
+        <label
+          for="agreement"
+        > I accept the term of the agreement
+          <input
+            type="checkbox"
+            id="agreement"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label
+          for="Email"
+          class="label"
+        > E-mail:
+          <input
+            type="email"
+            id="Email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label
+          for="Password"
+        > Password:
+          <input
+            type="password"
+            name="password"
+            id="Password"
+            minlength="8"
+            maxlength="32"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label
+          for="cats"
+          class="label"
+        > Do you love cats?
+          <input
+            type="radio"
+            value="yes"
+            name="cats"
+          > Yes
+          <input
+            type="radio"
+            value="No"
+            name="cats"
+          > No
+        </label>
+        <label
+          for="Color"
+          class="label"
+        > What is your favourite color?
+          <input
+            type="color"
+            name="color"
+            id="Color"
+          >
+        </label>
+        <label
+          for="sleepTime"
+          class="label"
+        > What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            id="sleepTime"
+          >
+        </label>
+        <label
+          for="Car"
+          class="label"
+        > What are your favorite brand of car?
+          <select
+            name="car"
+            id="Car"
+            multiple>
+            <option value="BMW">BMW</option>
+            <option value="AUDI">AUDI</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label
+          for="Rate"
+        > How do you rate our work?
+          <input
+            type="range"
+            name="rate"
+            id="Rate"
+            min="1"
+            max="10"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label
+          for="Comments"
+          class="label"
+        >Comments:
+          <textarea
+            name="comments"
+            id="Comments"
+            cols="20"
+            rows="2">
+          </textarea>
+        </label>
```

Comment:
```suggestion
            rows="2"
          ></textarea>
        </label>
```



https://user-images.githubusercontent.com/102413088/190588049-8321f2e0-3e6c-4d90-a524-d4aaff770512.mov



[виправ поведінку textarea](https://mate.academy/learn/css-advanced/forms#/video/885)